flocker/node/_loop.py

```diff
@@ -288,6 +291,10 @@ class ConvergenceLoop(object):
         ``None``.
 
     :ivar fsm: The finite state machine this is part of.
+
+    :ivar last_sent_local_state: Last reported local state to
+        control agent.
+    :type last_sent_local_state: tuple of IClusterStateChange
```

Comment:
It doesn't seem to be a tuple any more. Perhaps it was originally going to be a `2-tuple(state, client)` or something.
Document the `last_sent_client` attribute too.